toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -2073,6 +2079,32 @@ func setGrubCfgSELinux(grubPath string, kernelCommandline configuration.KernelCo
 	return
 }
 
+func setGrubCfgCGroup(grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
+	const (
+		cgroupPattern     = "{{.CGroup}}"
+		cgroupv1FlagValue = "systemd.unified_cgroup_hierarchy=0"
+		cgroupv2FlagValue = "systemd.unified_cgroup_hierarchy=1"
+	)
+	var cgroup string
```

Comment:
Where do we check if kernelCommandline.CGroup is valid or not? What's the value of the string cgroup when there is no match in the switch cases? Do we want to add a 'default' to the switch, and call sed only for the correct cgroup values? Thanks

Reply:
@neha170 the validity of CGroup entry is checked in the kernelcommandline.go file. The function is called isValid() and it gets utilized in the UnmarshalJSON function. If I'm understanding your second question correctly, I did add a default to the switch it is the last case, and it sets the value of string cgroup to "" as an empty string, so the sed call happening after the switch case is determined handles that as well. let me know if I'm misunderstanding something. Thanks!